src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static FruitStorageDao fruitStorageDao;
+    private static OperationHandler handler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitStorageDao = new FruitStorageDaoImpl();
+        handler = new BalanceOperationHandler(fruitStorageDao);
+    }
+
+    @Before
+    public void beforeEach() {
+        fruitStorageDao.getAll().clear();
+    }
+
+    @Test
+    public void balanceHandler_updateValidBalance_ok() {
```

Comment:
```suggestion
    public void handleOperation_updateValidBalance_ok() {
```

Reply:
you need to write a method that you test first, make the same with other similar cases